frontend/src/components/projectListView/ProjectListView.tsx

```diff
@@ -79,172 +58,79 @@ const ProjectListView = ({ tasks }: { tasks: Task[] }) => {
-              onChange={(e) => setOngoingDescription(e.target.value)}
-              inputProps={{
-                style: {
-                  padding: 5,
-                  background: 'white'
-                }
-              }}
-            />
-          </form>
-        ) : (
-          <AddCircleOutline
-            sx={{ cursor: 'pointer' }}
-            onClick={() => setOngoingEditing(true)}
-          />
-        )}
-      </div>
-      <ListTable listId="ongoing" tasks={ongoing} setTasks={setAllTasks} />
-      <div className="section-title">
-        <h4>Pending</h4>
-        {todoEditing ? (
-          <form
-            onSubmit={(e) => {
-              e.preventDefault();
-              addNewTask(setTodo, todoDescription, todo, 'todo');
-              setTodoEditing(false);
-              setTodoDescription('');
-            }}>
-            <TextField
-              placeholder="New Task Title"
-              variant="filled"
-              value={todoDescription}
-              onChange={(e) => setTodoDescription(e.target.value)}
-              inputProps={{
-                style: {
-                  padding: 5,
-                  background: 'white'
-                }
-              }}
-            />
-          </form>
-        ) : (
-          <AddCircleOutline
-            sx={{ cursor: 'pointer' }}
-            onClick={() => setTodoEditing(true)}
-          />
-        )}
-      </div>
-      <ListTable listId="to_do" tasks={todo} setTasks={setAllTasks} />
-      <div className="section-title">
-        <h4>Done</h4>
-        {doneEditing ? (
-          <form
-            onSubmit={(e) => {
-              e.preventDefault();
-              addNewTask(setDone, doneDescription, done, 'done');
-              setDoneEditing(false);
-              setDoneDescription('');
-            }}>
-            <TextField
-              placeholder="New Task Title"
-              variant="filled"
-              value={doneDescription}
-              onChange={(e) => setDoneDescription(e.target.value)}
-              inputProps={{
-                style: {
-                  padding: 5,
-                  background: 'white'
-                }
-              }}
-            />
-          </form>
-        ) : (
-          <AddCircleOutline
-            sx={{ cursor: 'pointer' }}
-            onClick={() => setDoneEditing(true)}
-          />
-        )}
-      </div>
-      <ListTable listId="done" tasks={done} setTasks={setAllTasks} />
+      {Object.values(categories).map((section) => (
+        <React.Fragment key={section}>
+          <div className='section-title'>
+            <h4>
+              {section === categories.documentation
+                ? 'Documentation'
+                : section === categories.ongoing
+                ? 'Ongoing'
+                : section === categories.todo
+                ? 'Pending'
+                : section === categories.done
+                ? 'Done'
+                : section}
```

Comment:
If you convert your categories map to an array and then have the values be the Title, then it would make this code much simpler